build/repo.targets

```diff
@@ -229,7 +229,14 @@
 
   <Target Name="SplitPackages">
     <ItemGroup>
-      <PackageArtifactFile Include="$(BuildDir)*.nupkg" Exclude="$(BuildDir)*.symbols.nupkg" />
+      <PackageSymbolsArtifactFile Include="$(BuildDir)*.symbols.nupkg" />
+    </ItemGroup>
+
+    <MakeDir Directories="$(ArtifactsDir)symbols\" />
+    <Move SourceFiles="@(PackageSymbolsArtifactFile)" DestinationFolder="$(ArtifactsDir)symbols\" />
```

Comment:
Sure we want to move instead of copy?

Reply:
Good question. What does the signing step expect?

Reply:
Well it doesn't really matter for signing since they look under the `.deps\symbols` directory. The question is do we want to keep the files under the original build directory. There's no harm either way.

Reply:
Okay maybe TC needs to unzip more files but that's not really an issue.

Reply:
One problem we have today is that the TC artifacts contains duplicate copies of our .nupkg files. I'm wondering if we should reduce duplication altogether by changing the splitpackages step into a move, not copy.

Reply:
The obvious downside is that we have to change downstream builds to be aware of all the subfolders in artifacts/. Today, we just assume everything is in artifacts/build

Reply:
Yea let's be consistent and choose one way or the other.

Reply:
Ok, for now, I'll make this a copy to stay consistent with dev and our current approach.